### PR TITLE
Add blui ci github workflow

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-5620.

#### Changes proposed in this Pull Request:

- add blui-ci github workflow
- NOTE - blui ci file has prettier & lint commented out and is not running (this needs setup using prettier from brightlayer-ui and there are a hand full of lint warnings)
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by manojleaton — APPROVED** [review]

File: .github/workflows/blui-ci.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn --immutable
+      - run: yarn docs-build --base= ${{env.base-href}} && touch ./dist/docs/.nojekyll
```
Comment (joebochill) [inline .github/workflows/blui-ci.yml:58]: is this space after base= going to cause a problem?

File: .github/workflows/blui-ci.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn
+      # - run: yarn docs-prettier
+      #   working-directory: docs
+    #   - run: yarn docs-lint
+    #     working-directory: docs
```
Comment (joebochill) [inline .github/workflows/blui-ci.yml:38]: We should be able to run lint and prettier right? We have the commands

Comment (joebochill) [reply .github/workflows/blui-ci.yml:38]: I saw your note in the PR description...can we log a bug for next sprint and assign it to @ArshdeepSingh3-eaton — this was supposed to be configured during the initial repository setup.

File: .github/workflows/blui-ci.yml (hunk 1 above)
Comment (JeffGreiner-eaton) [reply .github/workflows/blui-ci.yml:58]: it shouldn't, this is what vite build recommends. Here is what the index looks like in the build.

image

**Pull request merged**